src/vs/workbench/contrib/files/common/explorerModel.ts

```diff
@@ -242,6 +242,7 @@ export class ExplorerItem {
 			local.updateName(disk.name);
 		}
 		local._isDirectory = disk.isDirectory;
+		local._readonly = disk._readonly;
```

Comment:
In the end this change wasn't necessary to make this PR work, but it looked like an omission so I have left it in. @bpasero what do you think?